includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -276,7 +277,8 @@ public function create_intention(
 		$order_number,
 		$capture_method = 'automatic',
 		array $metadata = [],
-		$customer_id = null
+		$customer_id = null,
```

Comment:
I don't see any `create_intention()` calls that is passed by `$additional_parameters`. Do I miss anything?

Reply:
Good catch! It was used in a previous implementation and I forgot to revert it. Addressed in 2777b24